src/app/loginComponents/requests/requests.component.html

```diff
@@ -67,6 +66,32 @@ <h3>My Pending Requests</h3>
         </mat-card-content>
       </mat-card>
     </div>
+
+    <p class="pt-2">The following organizations that you have created are rejected.</p>
+    <mat-card class="alert alert-info" role="alert" *ngIf="(myRejectedOrganizationRequests$ | async)?.length == 0">
+      <mat-icon>info</mat-icon> You have no rejected organizations.
+    </mat-card>
+    <div fxLayout="column" fxLayoutGap="32px">
+      <mat-card *ngFor="let membership of (myRejectedOrganizationRequests$ | async); let i = index" [id]="'my-rejected-org-card-' + i">
+        <mat-card-header>
+          <mat-card-title>
+            <mat-icon color="primary" matTooltip="This organization has been rejected">warning</mat-icon>
+            <a [routerLink]="('/organizations/' + membership.organization.name )" [matTooltip]="membership?.organization?.name">{{membership.organization.displayName}}</a>
```

Comment:
Inconsistent on the use of `?`. I'm assuming the one after membership is unnecessary; if the one after organization is necessary, then it should be added  for the other usages.